components/istio-kyma-patch/README.md

```diff
@@ -0,0 +1,35 @@
+# Istio Kyma patch
+
+## Overview
+
+Application performs patch of already existing istio installation so kyma can be run.
+
+## Configuration
+
+Patch accepts following environmental variables:
+* `CONFIG_DIR` - directory where patches are placed. If not set script will look in directory it is placed in
+
+## Process
+
+Patch consists of several steps:
+1. Configuration of sidecar injector - istio-sidecar-injector configmap has following changes:
+    * `policy` is set to `disabled`
+    * `zipkinAddress` is changed so it points to zipkin deployed in kyma-system namespace
+    * all containers have set default `resources.limits.memory` to `128Mi` and `resources.limits.cpu` to `100m`
+
+1. Check if required CRDs are deployed on setup - script is looking for file named `required-crds`. In each line of this 
+file should contain one name of CRD which kyma requires to start.
+
+1. Istio components are patched - patch is expecting files in format `<resource-name>.<kind>.patch.json` which contain
```

Comment:
![dee](https://user-images.githubusercontent.com/35192450/46139110-f3010200-c24d-11e8-8835-5e1d1340ddf4.jpg)

This whole paragraph makes no sense. "Patch allows that resource to patch does not exist." ???? "Istio components are patched" - what does it mean that the components are patched. Please improve the sentences and make them more clear.